static/app/views/issueDetails/actions/shareModal.tsx

```diff
@@ -90,7 +90,9 @@ export default function ShareIssueModal({
                 checked={includeEventId}
                 onChange={() => setIncludeEventId(!includeEventId)}
               />
-              {t('Include Event ID in link')}
+              <span onClick={() => setIncludeEventId(!includeEventId)}>
```

Comment:
The preferred way to doing this is to make `CheckboxContainer` a `label`. Anything within the label will toggle the checkbox